### ISSUE TITLE
Select Component Types

### ISSUE THREAD

**Issue by kloehmann** [kloehmann/classifix#3]
## User Story
As a security engineer
I want to select component types for a specific system
in order to get only information relevant to my system

## Acceptance Criteria
GIVEN
Any checkbox for component types is unchecked
WHEN
the checkbox is checked in the browser
THEN
the entry is registered in the database

GIVEN
Any checkbox for component types is checked
WHEN
the checkbox is unchecked in the browser
THEN
the entry is dropped in the database

**Comment by OsamaAl-Wardi** [comment]
Dear Kevin,

I spent the last week and a half trying different approaches in to implement the checkbox system and relate it to the database. Martin couldn't help me the JS handler so I tried doing it the rails ways and it didn't work either. So for now I will go into the Technical characteristics part. I will try to resolve the issue with the checkboxes with you after your vacation.

Enjoy your vacation,
Osama Al-Wardi
